components/benefit_sponsors/app/models/benefit_sponsors/benefit_sponsorships/benefit_sponsorship.rb

```diff
@@ -410,7 +410,7 @@ def overlapping_coverage_exists?(benefit_application)
     end
 
     def oe_extended_applications
```

Comment:
This method isn't returning the extended applications as expected. We have the scenarios for cucumber set up properly, but the close open enrollment button is not appearing.